contracts/CryptoHamster/CryptoHamster.aes

```diff
@@ -3,13 +3,13 @@
 contract CryptoHamster =
 
     record state =
-        { index         : int
-        , map_hamsters  : map(string, hamster) }
+        { index         : int,
+        map_hamsters  : map(string, hamster) }
 
     record hamster =
-        { id    : int
-        , name  : string
-        , dna   : int }
+        { id    : int, 
+        name  : string, 
+        dna   : int }
```

Comment:
Any reasoning why do you insist on this formating?

Reply:
No particular reasoning other than having all the same format. Newbies get confused easily.